greenheart/tools/eco/hydrogen_mgmt.py

```diff
@@ -355,7 +356,7 @@ def run_h2_storage(
                 area_site,
                 mass_tank_empty_site,
                 _,
-            ) = h2_storage.distributed_storage_vessels(h2_capacity, 1)
+            ) = h2_storage.distributed_storage_vessels(greenheart_config["h2_capacity"], 1)
```

Comment:
Is there a reason to pull this from the yaml? I think it would make sense to make the capacity choices at the beginning of the function and then use that value.

Reply:
local variable, though we should save the storage capacity somewhere for output, but not in the config dict.

Reply:
basically leave as it was before and make sure the local variable is getting the right value